src/main/java/com/kodilla/ecommercee/CartController/CartController.java

```diff
@@ -0,0 +1,50 @@
+package com.kodilla.ecommercee.CartController;
+
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartDto;
+import com.kodilla.ecommercee.CartController.CartDomainDto.CartItemDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+@RestController
+@CrossOrigin("*")
+@RequestMapping(consumes = MediaType.APPLICATION_JSON_VALUE, value = "/carts")
+@RequiredArgsConstructor
+public class CartController {
+    
+    private static void createCart() {
+        List<CartItemDto> cart = new ArrayList<>();
```

Comment:
@PostMapping(value = "/createCart/{userId})
public ResponseEntity<Void> createCart(@PathVariable Long userId) 
It looks better, what do You think? it must take userId to know for which user we createCart